src/github-handlers/private-notifications/notification-message.ts

```diff
@@ -0,0 +1,71 @@
+export class NotificationMessage {
+  private static _formatMessage(template: string, data: Record<string, string>): string {
+    // "Parameter name `_` must match one of the following formats: strictCamelCase"
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    return template.replace(/{{\s*(\w+)\s*}}/g, (_, key) => data[key] || "");
```

Comment:
You shouldn't make exceptions to our lint rules

Reply:
https://github.com/ubiquity-os-marketplace/text-conversation-rewards/pull/285/files#diff-1dfacf90a90a9edf4a60df23849f5f73fbe71dfffa734847ed5a45297f3c6467R31

This is a hard one because first param is required but unused. `_` is supposed to stop the `unused vars` eslint error but your lint rules complain because `_` isn't `strictCamelCase`.

I either declare an unused var, ignore `naming-convention or create a generic `replace()` function for this usecase.

Reply:
I updated the `eslint` rules to allow the `no-unused-vars` override pattern to succeed and not be thrown by `naming-convention`

Reply:
Assuming the rule doesn't have other conflicts like what you identified, then we should also update the template with this.